homeassistant/components/teslemetry/strings.json

```diff
@@ -99,16 +120,76 @@
         "name": "Rear passenger window"
       },
       "vehicle_state_tpms_soft_warning_fl": {
-        "name": "Tire pressure warning front left"
+        "name": "TPMS warning front left"
       },
       "vehicle_state_tpms_soft_warning_fr": {
-        "name": "Tire pressure warning front right"
+        "name": "TPMS warning front right"
       },
       "vehicle_state_tpms_soft_warning_rl": {
-        "name": "Tire pressure warning rear left"
+        "name": "TPMS warning rear left"
       },
       "vehicle_state_tpms_soft_warning_rr": {
-        "name": "Tire pressure warning rear right"
+        "name": "TPMS warning rear right"
+      },
+      "pin_to_drive_enabled": {
+        "name": "Pin to drive enabled"
+      },
+      "drive_rail": {
+        "name": "Drive rail"
+      },
+      "driver_seat_belt": {
+        "name": "Driver seat belt"
+      },
+      "driver_seat_occupied": {
+        "name": "Driver seat occupied"
+      },
+      "passenger_seat_belt": {
+        "name": "Passenger seat belt"
+      },
+      "fast_charger_present": {
+        "name": "Fast charger present"
+      },
+      "gps_state": {
+        "name": "GPS state"
+      },
+      "guest_mode_enabled": {
+        "name": "Guest mode enabled"
+      },
+      "dc_dc_enable": {
+        "name": "DC DC"
```

Comment:
DC DC?

Reply:
Yeah, its what the value is called, I think its the DC to DC power converter

Reply:
Can we maybe make it more clear?

Reply:
Renamed to `DC to DC converter`